packages/xod-tabtest/src/TabtestValidator.re

```diff
@@ -0,0 +1,58 @@
+open Belt;
```

Comment:
Opening a lot of modules at the same time could lead to name clashes and confusion. I suggest using short aliases like `module B = Belt;`

Reply:
Although you’re right in general, `Belt` is kind of exception. I think it is OK to `open` the `Belt` root because it is stdlib and it matches the semi-official recommendation.

Other modules, yes, should be better aliased.